### DISCUSSION TITLE
Plugin in c#

### DISCUSSION THREAD

**Discussion by Eugeny335** [praydog/REFramework#1107] [General Questions]
I would like to make a plugin for RE4 in c#, but I can't find the dependencies "REFrameworkNET.dll ", I installed the Framework, but the folder "reframework/plugins/managed/dependencies" is empty, although the documentation says that files will automatically appear there. Could you tell me where I can get dependencies, what to use c# as a language to develop a plugin for the RE4 game and use the API?

**Comment by praydog** [comment]
Use https://github.com/praydog/REFramework/actions

Look for builds with `csharp-api` next to them.

**Reply by Eugeny335** [reply]
Thanks

**Reply by Eugeny335** [reply]
Why am I installing dinput8, but my reframework/plugins folder is empty? I can't automatically generate RE Framework NET.*.dll dependencies, can you tell me how to generate them?
![image](https://github.com/user-attachments/assets/e9b671d3-54c0-42ef-a4e0-3fd9f18342be)

**Reply by praydog** [reply]
https://github.com/praydog/REFramework/actions/runs/10024822253

You need to extract the entire zip file. It comes with the .NET API inside `reframework/plugins`
